homeassistant/components/melnor/switch.py

```diff
@@ -41,7 +41,6 @@ class MelnorSwitchEntityDescription(
         device_class=SwitchDeviceClass.SWITCH,
         icon="mdi:sprinkler",
         key="manual",
-        name="Manual",
```

Comment:
With the new naming system we don't need a default name. The switch inherits the device name. Future switches will have their own naming suffixes to differentiate them but the default switch will always be this one as it controls the on/off state of the valve.